test/e2e/virtual/initializingworkspaces/virtualworkspace_test.go

```diff
@@ -115,7 +117,7 @@ func TestInitializingWorkspacesVirtualWorkspaceAccess(t *testing.T) {
 	sourceKcpClusterClient, err := kcpclient.NewClusterForConfig(sourceConfig)
 	require.NoError(t, err)
 
-	kubeClusterClient, err := kubernetes.NewClusterForConfig(sourceConfig)
+	kubeClusterClient, err := kubernetes.NewForConfig(serverutil.NewClusterConfig(sourceConfig))
```

Comment:
Either we could wrap like this, or if this seems convoluted we could have:
```go
clusterCfg := serverutil.NewClusterConfig(sourceConfig)
```

And have `clusterCfg` as a convention for round tripper wrapped ones.
This way our convention would be:
`cfg`/ `sourceCfg` / `DefaultCfg` - refer to existing configs as is.
`clusterCfg` - refers to any of the above mentioned configs wrapped with round tripper.

Does that sound fine?
cc: @stevekuznetsov

Reply:
:+1: